packages/devp2p/src/rlpx/peer.ts

```diff
@@ -52,7 +52,7 @@ export type HelloMsg = {
 }
 
 export interface ProtocolDescriptor {
-  protocol?: any
+  protocol: any
```

Comment:
I guess this is ok, there is always a protocol present (otherwise the whole thing makes no sense).